### ISSUE TITLE
Github, overlooked elements in Filter+

### ISSUE THREAD

**Issue by NSExceptional** [darkreader/darkreader#904]
- [ ] [The search bar](https://i.imgur.com/Xv7SReJ.png)
- [ ] Buttons in general would do nicely to have their gradients reversed/swapped if that's possible.
- [ ] [`btn-primary`](https://i.imgur.com/jryYj6Q.png), both selected and unselected. Too bright, and text needs to be white.
- [ ] Similarly, [this thing](https://i.imgur.com/l7fe8Hm.png), class `State State--green`
- [ ] [This menu on my issue](https://i.imgur.com/CwMgId9.png), text when mousing over should still be white

Will add more as I find them

**Comment by AndrewDaGuy** [comment]
Related to #4182

**Comment by Gusted** [comment]
> Related to #4182

Not at all. #4182 was specifically created due to the new design of Github and dark reader not having the right logic.

**Comment by AndrewDaGuy** [comment]
> > Related to #4182
> 
> Not at all. #4182 was specifically created due to the new design of Github and dark reader not having the right logic.

Didn't read the title properly 😔 thought I saw dynamic instead of filter+
